Index: src/ru/baranov/task8/num4/NumberForth.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/ru/baranov/task8/num4/NumberForth.java	(revision 76a3a506357819cdcf1ca9b2afb14b03e043377f)
+++ src/ru/baranov/task8/num4/NumberForth.java	(date 1586882813687)
@@ -9,7 +9,7 @@
         Contract a = new Contract();
         a.c();
 
-        Statement d = new Statement(a.getNumber(), a.getDate(), a.getContain());
+        Statement d = new Statement(a.getNumber(), a.getDate(), a.getContain());//КОНВЕРТЕР
         d.st();
     }
 }
Index: src/ru/baranov/task8/num3/Count.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/ru/baranov/task8/num3/Count.java	(revision 76a3a506357819cdcf1ca9b2afb14b03e043377f)
+++ src/ru/baranov/task8/num3/Count.java	(date 1586882813699)
@@ -3,28 +3,28 @@
 public class Count {
 
 
-    static int num_objects = 0;
+    private static int numObjects = 0;
 
     {
     }
 
     public Count() {
-        num_objects += 1;
+        numObjects += 1;
     }
 
     public Count(int n) {
-        num_objects += 1;
+        this();
     }
 
     public Count(String s) {
-        num_objects += 1;
+        this();
     }
 
     public Count(int g, String d) {
-        num_objects += 1;
+        this();
     }
 
     public void Info() {
-        System.out.println("Количество создаваемых объектов: " + num_objects);                                        //метод для получения информации о количестве
+        System.out.println("Количество создаваемых объектов: " + numObjects);                                        //метод для получения информации о количестве
     }
 }
Index: src/ru/baranov/task9/num1/Animal.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/ru/baranov/task9/num1/Animal.java	(revision 76a3a506357819cdcf1ca9b2afb14b03e043377f)
+++ src/ru/baranov/task9/num1/Animal.java	(date 1586882813694)
@@ -5,7 +5,7 @@
 
     public String getName() {
         return name;
-    }
+    }//ABSTRACT
 
     public Animal(String name) {
         this.name = name;
